api/control/v1alpha1/types.go

```diff
@@ -268,11 +268,15 @@ type BindingList struct {
 // +genclient:nonNamespaced
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Cluster,shortName={ct},categories={all}
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="WGROUP",type="string",JSONPath=".spec.apiGroup"
```

Comment:
What's the W stands for ?

Reply:
"Workload". Something is needed to distinguish from the object's own attributes.

Reply:
And, can we use RESOURCE instead of WRSC ? We do not have a lot of columns and perhaps clarity is better than saving columns space.

Reply:
I think it is confusing to have columns labeled "GROUP" and "RESOURCE" that do **NOT** refer to the object's own GROUP and RESOURCE. Remember that these objects do indeed have their own GROUP and RESOURCE. That is why I think we need some distinguishing information in the column headers.

Reply:
What do you think of making these columns say "SUBJECT_GROUP" and "SUBJECT_RESOURCE"?

Reply:
or "WORKLOAD_GROUP" and "WORKLOAD_RESOURCE"?

Reply:
I would prefer "SUBJECT_GROUP" and "SUBJECT_RESOURCE". WORKLODAD_ is fine but we may get stuck debating if an object to deliver is a workload or a configuration.

Reply:
@vMaroon : can you live with "SUBJECT_GROUP" and "SUBJECT_RESOURCE"?

Reply:
Yes.